src/App.tsx

```diff
@@ -33,63 +54,19 @@ export const App: React.FC = () => {
           items per page
         </label>
       </div>
+      <Pagination
+        total={items}
+        perPage={perPage}
+        currentPage={currentPage}
+        onPageChange={onPageChang}
```

Comment:
```suggestion
        onPageChange={setCurrentPage}
```
we can use it directly